client/my-sites/backup/backup-storage-space/backup-storage-space-upsell/style.scss

```diff
@@ -0,0 +1,42 @@
+.backup-storage-space-upsell__title-text {
+	font-size: 1rem;
+	font-weight: bold;
+	margin-bottom: 16px;
+}
+
+a.backup-storage-space-upsell__button {
+	color: #1e1e1e;
+}
+
+.backup-storage-space-upsell__button {
+	border: 2px solid var( --studio-jetpack-green );
+	border-radius: 2px;
+	padding: 16px 24px;
+	width: 100%;
+	height: 100%;
+	font-size: 1rem;
+
+	&:hover {
+		background: var( --color-accent-0 );
+	}
+}
+
+.backup-storage-space-upsell__grid {
+	display: grid;
+	grid-template-columns: auto min-content;
+	align-items: center;
+	width: 100%;
+	text-align: left;
+
+	.gridicons-arrow-right {
+		color: var( --studio-jetpack-green );
+	}
+}
```

Comment:
Could using `grid` here (and over in `__progress-bar-container`) be slightly overkill for the situation, since we're only dealing with one row/column? I wonder if `flex` could do the trick. But this is just me musing, not a critical issue by any means.

Reply:
In this case there's actually two columns, one for the text and one for the arrow. I wanted the arrow column to shrink to the content size and the text column to take up the rest of the space, which seemed good for a grid. However I think it could be done with `flex` by setting `flex-grow: 1` on the arrow and having no `flex-grow` on the text, but that seems less obvious to me.